tests/utils.go

```diff
@@ -1772,6 +1772,14 @@ func NewRandomVMIWithe1000NetworkInterface() *v1.VirtualMachineInstance {
 	return vmi
 }
 
+func NewRandomVMIWithInvalidNetworkInterface() *v1.VirtualMachineInstance {
```

Comment:
I guess this function can be in the test file and not in utils.

Reply:
I put this function here as a "sister" function to NewRandomVMIWithe1000NetworkInterface(), which is also implemented here although it is only used in vmi_networking_test.go.

Reply:
@yossisegev you can do something like this directly in the test.
```golang
vmi := NewRandomVMIWithe1000NetworkInterface()
vmi.Spec.Domain.Devices.Interfaces[0].Model = "gibberish"
_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
Expect(err).To(HaveOccurred())
```

Reply:
@SchSeba , I'll go with with moving this function to the test file itself. Using NewRandomVMIWithe1000NetworkInterface() seems out of context of this specific test.

Reply:
sounds good